server/controllers/userController.js

```diff
@@ -2,19 +2,53 @@ const User = require("../models/userModel");
 
 const userController = {};
 
+/* Check to see if a user exists by email and username in the database on signup */
+userController.verifyUser = (req, res, next) => {
+  // pull out the username and password from the body of the request
+  const { username, password } = req.body;
+
+  // check the database to find all users that match the given data
+  User.find({ username, password })
+    .exec()
+    .then(userData => {
+      // set isVerified flag to false initially
+      let isVerified = false;
+
+      // if the the specific username and password are found in database, set isVerified to true
+      if (
+        userData[0].username === username &&
+        userData[0].password === password
+      ) {
+        isVerified = true;
+        res.locals.isVerified = isVerified;
+
+        console.log("User successfully verified!");
+        return next();
+      }
+      next();
```

Comment:
need to remove the extra next

Reply:
Changed it to 'return next', keeping it there to send res.locals.isVerified = false to frontend